Extension/src/LanguageServer/references.ts

```diff
@@ -420,9 +420,10 @@ export class ReferencesManager {
                 if (!foundUnconfirmed) {
                     this.resultsCallback(referencesResult, true);
                 } else {
+                    this.resultsCallback(null, true);

```

Comment:
>this.resultsCallback(null, true); [](start = 20, length = 33)

Add comment for calling this.resultsCallback(null, true) ? Does this cancel the progress bar?